pkg/config-disk/config-disk.go

```diff
@@ -72,6 +72,7 @@ func (c *configDiskClient) Define(vm *v1.VirtualMachine) (bool, error) {
 
 	jobKey := createKey(vm)
 
+	// FIXME support re-creating the disks if the VM is down and the data changed
```

Comment:
This already happens.

Reply:
Are you sure? For me it looks like, that only if the VM is re-created, the disk is updated, but not if e.g. the VM crashed and the metadata changed in the meantime. Wouldn't it then just re-started with the disk containing the old data?